apache2/waf_logging/waf_log_util_internal.h

```diff
@@ -1,9 +1,62 @@
 #ifndef _WAF_LOG_UTIL_INTERNAL_HEADER
 #define _WAF_LOG_UTIL_INTERNAL_HEADER
 
-#include <google/protobuf/util/json_util.h>
-#include"waf_format.pb.h"
 #include <fstream>
+#include <unordered_map>
```

Comment:
Doesn't seem we use `unordered_map` below in this file - let's remove this include.